python/paddle/nn/functional/distance.py

```diff
@@ -106,3 +106,41 @@ def pairwise_distance(x, y, p=2.0, epsilon=1e-6, keepdim=False, name=None):
         )
 
         return out
+
+
+def pdist(x, p=2.0, name=None):
+    r'''
+    Computes the p-norm distance between every pair of row vectors in the input.
+
+    Args:
+        x (Tensor): The input tensor with shape :math:`N \times M`.
+        p (float, optional): The value for the p-norm distance to calculate between each vector pair. Default: :math:`2.0`.
+        name (str, optional): For details, please refer to :ref:`api_guide_Name`. Generally, no setting is required. Default: None.
+
+    Returns:
+        Tensor with shape: :math:`N(N-1)/2` the dtype is same as input tensor.
```

Comment:
```suggestion
        Tensor with shape :math:`N(N-1)/2` , the dtype is same as input tensor.
```
`Returns` 的内容加冒号的话，冒号前的部分会解析成 returns type
<img width="393" alt="image" src="https://github.com/PaddlePaddle/Paddle/assets/70642955/e7e830a8-0541-41c5-8804-8e1ee934d0a5">